### ISSUE TITLE
Clip area of interest to custom polygon

### ISSUE THREAD

**Issue by jome1** [csdms/bmi-topography#76]
Is it right, that it is only possible to crop the data according to a bounding box?

I would love a functionality that clips the data to a polygon. So, I for example only download the data for a federal state but not the bbox around this federal state.

**Comment by mdpiper** [comment]
Yes, the [OpenTopography API](https://portal.opentopography.org/apidocs/) we wrap only allows a bounding box, unfortunately.
